user/views.py

```diff
@@ -2,0 +1,23 @@
+from rest_framework import generics, permissions
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.settings import api_settings
+
```

Comment:
There are multiple blank lines here. It's better to keep your code consistent. Delete line 7 and 8, leaving only one blank line.